go/tools/builders/stdliblist_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+)
+
+func Test_stdliblist(t *testing.T) {
+	testDir := t.TempDir()
+	f, _ := ioutil.TempFile(testDir, "*")
```

Comment:
You don't need to create a temp file in a temp dir. `outJSON = filepath.Join(testDir, "out.json")` should be enough

Reply:
done